docs/api/proxy-based-tracking.md

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
This file is not currently added to the Docusaurus sidebar, and so it's not showing up in the docs.

Having said that: for now, let's put the content into the existing "Hooks" docs page.  Longer-term, we might want to consider breaking that page into separate pages per hook, but let's just keep it all in one page at the moment.